packages/api-derive/test/e2e/promise.spec.ts

```diff
@@ -100,4 +103,35 @@ describe.skip('derive e2e', () => {
       done();
     });
   });
+
+  describe.only('verifies derive.staking.unlocking',() => {
+    const BOND_VALUE = 10;
+    const UNBOND_VALUE = 1;
+    const ALICE_STASH = '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY';
+    const ALICE = '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY';
+    const keyring = testKeyring();
+    const aliceStashPair = keyring.getPair(ALICE_STASH);
+    const alicePair = keyring.getPair(ALICE);
+
+    it('bondsExtra funds for Alice Stash', () => {
+      return api.tx.staking.bondExtra(BOND_VALUE)
+        .signAndSend(aliceStashPair, ({ events = [], status }) => {
+          expect(status.type).toMatch(/Ready/i);
```

Comment:
You actually want to wait for finalised before it is "ok"

Reply:
Thanks, totally missed that, not familiar with the `status` yet